pkg/envoy/ads/server.go

```diff
@@ -32,6 +32,9 @@ const (
 
 	// workerPoolSize is the default number of workerpool workers (0 is GOMAXPROCS)
 	workerPoolSize = 0
+
+	// xdsServerCertificateCommonName is the name of the certificate for the ADS server
+	xdsServerCertificateCommonName = "ads"
```

Comment:
The variable name indicates this being a CN, while the comment refers to this as the certificate name. Could you make the terminology consistent.